source/.gitignore

```diff
@@ -113,3 +113,9 @@ Thumbs.db
 /ios/xcode/native/
 /ios/IOSLauncher.app
 /ios/IOSLauncher.app.dSYM
+
+## Auto-generated .log files
+*.log
+
+## save file, shouldn't be tracked in repo
+save.json
```

Comment:
Right now the save.json file won't be created if it doesn't exist, so it needs to be in the repo but shouldn't be tracked. Not sure if this could cause issues